### PR TITLE
Add metric endpoint

### PR DESCRIPTION
**Pull request by Ne184**
Added feedback functionality from the user and also metric endpoint for Prometheus.
The branch is based on the integrate-kubernetes branch

### CODE REVIEW

File: k8s/app.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
```
Comment (thijsnulle) [inline k8s/app.yml]: Is this file needed as we want to start it from the `operation` repo?

**Review by thijsnulle — APPROVED** [review]

File: k8s/app.yml (hunk 1 above)
Comment (Ne184) [reply k8s/app.yml]: Yeah, I think we do not need it here, will remove it

**Review by danielatoader — APPROVED** [review]
